### PR TITLE
Fix rgb server

### PR DESCRIPTION
**Pull request by SeptimiuVana**

### CODE REVIEW

File: examples/aditof-demo/aditofdemocontroller.cpp [hunk 1]
```diff
@@ -52,7 +52,7 @@ AdiTofDemoController::AdiTofDemoController()
             LOG(WARNING) << "no frame type available!";
             return;
         }
-        camera->setFrameType(frameTypes.front());
+        //camera->setFrameType(frameTypes.front());
```
Comment (acozma) [inline examples/aditof-demo/aditofdemocontroller.cpp:55]: Remove this comment

File: examples/aditof-demo/aditofdemocontroller.cpp [hunk 2]
```diff
@@ -84,7 +84,7 @@ bool AdiTofDemoController::setRegularConnection() {
             LOG(WARNING) << "no frame type available!";
             return false;
         }
-        camera->setFrameType(frameTypes.front());
+        //camera->setFrameType(frameTypes.front());
```
Comment (acozma) [inline examples/aditof-demo/aditofdemocontroller.cpp:87]: Remove this line

File: examples/aditof-demo/aditofdemocontroller.cpp [hunk 3]
```diff
@@ -122,7 +122,7 @@ bool AdiTofDemoController::setNetworkConnection(const std::string &ip) {
...
```
Comment (acozma) [inline examples/aditof-demo/aditofdemocontroller.cpp:125]: Remove this line

File: examples/aditof-demo/aditofdemoview.cpp [hunk 4]
```diff
@@ -1100,7 +1112,7 @@ void AdiTofDemoView::_displayIrImage() {
 
         m_irImage = cv::Mat(frameHeight, frameWidth, CV_16UC1, irData);
         m_irImage.convertTo(m_irImage, CV_8U, 255.0 / max_value_of_IR_pixel);
-        flip(m_irImage, m_irImage, 1);
+        flip(m_irImage, m_irImage, 0);
```
Comment (acozma) [inline examples/aditof-demo/aditofdemoview.cpp:1115]: Should flip on the horizontal axis only if SMART_3D is defined
Do not flit on the vertical axis anymore

File: examples/aditof-demo/aditofdemoview.cpp [hunk 5]
```diff
@@ -1129,7 +1141,7 @@ void AdiTofDemoView::_displayBlendedImage() {
 
     m_irImage = cv::Mat(frameHeight, frameWidth, CV_16UC1, irData);
     m_irImage.convertTo(m_irImage, CV_8U, 255.0 / max_value_of_IR_pixel);
-    flip(m_irImage, m_irImage, 1);
+    flip(m_irImage, m_irImage, 0);
```
Comment (acozma) [inline examples/aditof-demo/aditofdemoview.cpp:1144]: Should flip on the horizontal axis only if SMART_3D is defined
Do not flit on the vertical axis anymore

File: sdk/src/cameras/3d-smart-camera/camera_3d_smart.cpp [hunk 6]
```diff
@@ -427,9 +423,9 @@ aditof::Status Camera3D_Smart::setFrameType(const std::string &frameType) {
     if (!m_devStarted) {
         status = m_depthSensor->start();
         status = m_rgbSensor->start();
-        if (status != Status::OK) {
-            return status;
-        }
+        //if (status != Status::OK) {
```
Comment (acozma) [inline sdk/src/cameras/3d-smart-camera/camera_3d_smart.cpp:426]: Remove these lines

File: examples/aditof-demo/aditofdemoview.cpp [hunk 7]
```diff
@@ -1160,23 +1172,25 @@ void AdiTofDemoView::_displayRgbImage() {
...
+        localFrame->getDetails(frameDetails);
 
-        //aditof::FrameDetails frameDetails;
-        //localFrame->getDetails(frameDetails);
+        int frameHeight = static_cast<int>(frameDetails.rgbHeight);
+        int frameWidth = static_cast<int>(frameDetails.rgbWidth);
+        int max_value_of_IR_pixel = 32768;
+        m_rgbImage = cv::Mat(frameHeight, frameWidth, CV_8UC3, rgbData);
+        cv::cvtColor(m_rgbImage, m_rgbImage, cv::COLOR_RGB2BGR);
+        cv::resize(m_rgbImage, m_rgbImage, cv::Size(640, 480));
```
Comment (acozma) [inline examples/aditof-demo/aditofdemoview.cpp:1189]: Resize to 960 x 540

File: examples/aditof-demo/aditofdemoview.cpp [hunk 8]
```diff
@@ -1160,23 +1172,25 @@ void AdiTofDemoView::_displayRgbImage() {
...
+        m_rgbImage *= 4;
+        //m_rgbImage += cv::Scalar(50, 50, 50);
```
[the remark this answers is not in the document]
Comment (SeptimiuVana) [reply examples/aditof-demo/aditofdemoview.cpp]: Fixed

**Review by acozma — APPROVED** [review]

**Pull request merged**